Bottom: 91851fc2fbedb12792100d7a87852db7ed8fb3cd
Top:    ee2727d3b725f3c0c755c85c3eb5898b8f44a4b7
Author: Petr Vileta <fidokomik@gmail.com>
Date:   2013-07-17 00:39:39 +0200

Update script.js

Functions max() and min() fixed.

---

diff --git a/script.js b/script.js
index 9278864..74fe026 100644
--- a/script.js
+++ b/script.js
@@ -53,10 +53,10 @@ function average(a) {
 }
 
 function min(a) {
-	var s=a[0];
+	var s=1*a[0];
 	for (var i=1;i<a.length;i++) {
-		if (a[i]<s) {
-			s=a[i];
+		if (1*a[i]<s) {
+			s=1*a[i];
 		}
 	}
 	return s;
@@ -64,10 +64,10 @@ function min(a) {
 
 
 function max(a) {
-	var s=a[0];
+	var s=1*a[0];
 	for (var i=1;i<a.length;i++) {
-		if (a[i]>s) {
-			s=a[i];
+		if (1*a[i]>s) {
+			s=1*a[i];
 		}
 	}
 	return s;
